task/deploy-uniswapv3.js

```diff
@@ -6,18 +6,26 @@ const { task } = require('hardhat/config')
 
 task('deploy-mock-uniswap', 'deploy mock uniswapv3')
 .setAction(async (args, hre) => {
+  let netname = args.netname ? args.netname : hre.network.name.replace('_fork', '')
   const [ signer ]  = await hre.ethers.getSigners()
   const weth_pack = args.weth_pack
-  const pack = require('../pack/uniswapv3_ethereum.dpack.json')  // reference deployment for mocks
+  const pack = require(`../pack/uniswapv3_${netname}.dpack.json`)  // reference deployment for mocks
+  return pack
```

Comment:
Is this meant to be like this? Previously this task was for creating a mock version for a non fork test